src/main/java/org/sonatype/nexus/ci/config/Nxrm3Configuration.groovy

```diff
@@ -69,5 +70,36 @@ class Nxrm3Configuration
         error(e, 'Nexus Repository Manager 3.x connection failed')
       }
     }
+
+    @Override
+    FormValidation doCheckServerUrl(@QueryParameter String value) {
+      def validation = super.doCheckServerUrl(value)
+
+      if (validation.kind != OK) {
+        return validation
+      }
+
+      // check nexus version, warn if < 3.13.0 PRO
+      try {
+        def response = new XmlSlurper().parseText(new URL("${value}/service/rest/wonderland/status").text)
+        def edition = response.edition.text()
+        def version = response.version.text()
+        def (major, minor) = version.tokenize('.').take(2).collect { it as int }
+
+        if (!edition.equalsIgnoreCase('pro') || major < 3 || minor < 13) {
+          return FormValidation.warning(
+              "NXRM ${edition} ${version} found. Some operations require a Professional Edition Nexus Repository " +
+                  "Manager server of version 3.13.0 or newer; use of an incompatible server will result in failed " +
+                  "builds")
+        }
+      }
+      catch (Exception e) {
+        return FormValidation.warning(
```

Comment:
aren't we burying info here without logging the exception? I'm not familiar with how Jenkins will treat this so maybe it's being logged automatically but otherwise this looks like it will complicate support scenarios